doc/source/serve.rst

```diff
@@ -0,0 +1,17 @@
+Ray Serve (Experimental)
+========================
+
+Ray Serve is a serving library that exposes python function/classes to HTTP.
+It has built-in support for flexible traffic policy. This means you can easy
+split incoming traffic to multiple implementations. 
+
+With Ray Serve, you can deploy your services at any scale.
```

Comment:
Are you only planning to support 2.7?

Reply:
Python 3 and up, have error on import. Warning added 
![image](https://user-images.githubusercontent.com/21118851/66167892-60c5d380-e5f0-11e9-8aa7-fae423fcd4eb.png)